pinot-tools/src/main/java/org/apache/pinot/tools/data/generator/NumberGenerator.java

```diff
@@ -61,23 +66,23 @@ public void init() {
         final int start = rand.nextInt(cardinality);
         final int end = start + cardinality;
         for (int i = start; i < end; i++) {
-          intValues.add(new Integer(i));
+          intValues.add(i);
         }
         break;
       case LONG:
         longValues = new ArrayList<Long>();
         final long longStart = rand.nextInt(cardinality);
         final long longEnd = longStart + cardinality;
         for (long i = longStart; i < longEnd; i++) {
-          longValues.add(new Long(i));
+          longValues.add(i);
         }
         break;
       case FLOAT:
         floatValues = new ArrayList<Float>();
-        final float floatStart = rand.nextFloat() * rand.nextInt(1000);
+        final float floatStart = Math.round(rand.nextFloat()* 100.0f) / 100.0f;
```

Comment:
why this change?

Reply:
This makes the generated float numbers easier on the eye - with two decimal places.

Reply:
Got it. Thanks